src/pages/_document.js

```diff
@@ -3,10 +3,28 @@ import Document, { Head, Main, NextScript } from 'next/document'
 
 export default class DefaultDocument extends Document {
   render() {
+    const FB_SCRIPT = `
+      window.fbAsyncInit = function() {
+        FB.init({
+          appId      : '240115020081725',
+          xfbml      : true,
+          version    : 'v3.0'
+        });
+        FB.AppEvents.logPageView();
+      };
+      (function(d, s, id) {
+        var js, fjs = d.getElementsByTagName(s)[0];
+        if (d.getElementById(id)) return;
+        js = d.createElement(s); js.id = id;
+        js.src = 'https://connect.facebook.net/pt_BR/sdk.js#xfbml=1&version=v3.0&appId=240115020081725&autoLogAppEvents=1';
+        fjs.parentNode.insertBefore(js, fjs);
+      }(document, 'script', 'facebook-jssdk'));
+      `
     return (
       <html lang="en">
         <Head>
           <link rel="stylesheet" href="/_next/static/style.css" />
+          <script dangerouslySetInnerHTML={{ __html: FB_SCRIPT }} />
```

Comment:
Maybe we can also make these social logins as a bool variable in the env vars, if someone wants to use this auth component, but not the social login we don't need to inject this script in the page and can hide the buttons, what you think?

Reply:
I think it's a great ideia although it's not possible to set a boolean on env.var just strings